plugins/generator-1.19.4/forge-1.19.4/generator.yaml

```diff
@@ -62,7 +62,7 @@ base_templates:
     writer: file
     name: "@RESROOT/META-INF/accesstransformer.cfg"
     condition_any:
-      - ${w.hasElementsOfType('feature')}
+      - ${w.hasElementsOfType('feature') || w.hasElementsOfType('livingentity')}

```

Comment:
Since this is condition any, it is better to list each condition in each line to execute OR

Reply:
Done!